src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -207,6 +207,11 @@ internal static readonly HashSet<(string, string)> DefaultHistogramBoundLongMapp
     /// </summary>
     public string MeterVersion => this.InstrumentIdentity.MeterVersion;
 
+    /// <summary>
+    /// Gets the attributes (tags) for the metric stream.
+    /// </summary>
+    public IEnumerable<KeyValuePair<string, object?>> MeterTags => this.InstrumentIdentity.MeterTags;
```

Comment:
Are meter tags expected to be unaffected by Views?

Reply:
yes. just like Meter name and version.

Reply:
Is this coming from the spec? We are exporting Meter tags are regular tags as well, right?

Reply:
This is part of spec.
Meter tags are separate from regular tags. Metertags are not part of aggregation. It is at the same level as Meter.Name,Version, forming the "instrumentation scope" from the spec.

Reply:
Or is it up to the exporter/backend to decide how it wants to treat them? In any case, there should be an official stance documented for how [this](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/metrics/customizing-the-sdk#select-specific-tags) affects or does not affect Meter tags.

Reply:
mm.. The name "tags/attributes" is what is confusing? There are tags/attributes on Resource level, then again on Meter level, and then on each recording - all are independent. The ones on recording are used for Aggregationg, the ones on meter are used to populate InstrumentaionScope, and one on Resource are used to.. well populate Resource...

Do we need to do anything about this? I can think of mentioning this in our best-practices doc so users can pick the right choice based on the needs and avoid mistakes/less-ideal solutions like [this](https://github.com/open-telemetry/opentelemetry-dotnet/discussions/5088).

(I intentionally did not mention anything about the Instrument level tags .NET 8 added... That is not part of spec, and we'll need to discuss how to deal with that later. It could be merged with with Meter tags (i.e won't affect aggreagation) OR merged with Tags provided at measurement time (affecting aggregation)...That would make things more interesting , lets worry about it later.

Reply:
I discussed this awhile back with @jack-berg from Java. The conclusion we came to is that it would be incorrect for views to have any affect over scope attributes.

Somewhat similarly, views do not affect resource attributes, and for good reason because if a view could affect resource attributes this would open the door for an SDK instance to emit multiple distinct resources which is not allowed by the spec.

More generally and with regards to scope attributes, the attribute keys defined by a view only govern the attributes supplied by individual measurements - i.e., the attributes influencing aggregation as @cijothomas notes.

Reply:
> I intentionally did not mention anything about the Instrument level tags .NET 8 added...

This is essentially "bound" instruments and to my knowledge has gotten some renewed interest for reintroducing in the spec.

Reply:
It isn't a blocker for me but I would personally feel better if the spec was clarified to say something like: "Views are NOT able to select/drop attributes inherited via Resource or Instrumentation Scope."

Resource users can control. But instrumentation scope you are kind of at the mercy of the author to not add a bunch of noise. If they do, you will have to pay for it 💰 For that reason I can imagine the spec one day becoming opinionated about how to drop stuff added through the instrumentation scope (may or may not be through the View API or something else).

Reply:
> If they do, you will have to pay for it 

That really depends on the backend! Good chance most backends are just dropping it today (given its a new feature). Or could be opt-in basis as well! The only cost is the cost of serializing+network - but instrumentation scope is not repeated for each metricpoint, so not that bad.

Reply:
> > I intentionally did not mention anything about the Instrument level tags .NET 8 added...
> 
> This is essentially "bound" instruments and to my knowledge has gotten some renewed interest for reintroducing in the spec.

The topic of bound instrument is precisely why I started looking into this! 😄 (OTel Rust has solid need for that, I was trying to prototype how it'd look in a more familiar code-base)

Reply:
All okay to resolve this? 
I will sent a follow up for update metric best practices to talk about Meter.Tags, and also highlight the importance of leveraging Resources more. 
Not planning to do any doc change for the View part.